homeassistant/components/telegram_bot/__init__.py

```diff
@@ -426,6 +440,8 @@ async def async_send_telegram_message(service: ServiceCall) -> ServiceResponse:
             await notify_service.set_message_reaction(context=service.context, **kwargs)
         elif msgtype == SERVICE_EDIT_MESSAGE_MEDIA:
             await notify_service.edit_message_media(context=service.context, **kwargs)
+        elif msgtype == SERVICE_DOWNLOAD_FILE:
+            return await notify_service.download_file(context=service.context, **kwargs)
```

Comment:
I think we need to mark this action as (always) having a return value.

Reply:
https://developers.home-assistant.io/docs/dev_101_services/#supporting-response-data

Reply:
@MartinHjelmare I pushed it with `ONLY` for now